crates/steel-derive/src/lib.rs

```diff
@@ -39,26 +39,37 @@ fn parse_key_value_pairs(args: &Punctuated<Meta, Token![,]>) -> HashMap<String,
         if let Meta::NameValue(n) = nested_meta {
             let key = n.path.get_ident().unwrap().to_string();
 
-            match &n.value {
-                Expr::Lit(ExprLit {
-                    lit: Lit::Str(s), ..
-                }) => {
-                    map.insert(key, s.value());
-                }
-                Expr::Lit(ExprLit {
-                    lit: Lit::Bool(b), ..
-                }) => {
-                    map.insert(key, b.value().to_string());
+            let mut value = &n.value;
+
+            loop {
+                match value {
+                    Expr::Lit(ExprLit {
+                        lit: Lit::Str(s), ..
+                    }) => {
+                        map.insert(key, s.value());
+                        break;
+                    }
+                    Expr::Lit(ExprLit {
+                        lit: Lit::Bool(b), ..
+                    }) => {
+                        map.insert(key, b.value().to_string());
+                        break;
+                    }
+                    Expr::Group(ExprGroup { expr, .. }) => {
+                        value = &**expr;
+                    }
+                    _ => {}
```

Comment:
Sorry, just noticing this - will this loop infinitely if there isn't a match? e.g. you don't use either a literal or a group?

Reply:
Oops, good catch